Parsers/ASimAuditEvent/Tests/Barracuda_WAF_ASimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,98 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUsername]"
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
```

Comment:
Reason in EventMessage should be mapped to EventResultDetails.

Reply:
Done